packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -2471,7 +2472,6 @@ interface IChoiceGroupOptionProps extends IChoiceGroupOption {
   onChange?: OnChangeCallback;
   onFocus?: OnFocusCallback;
   required?: boolean;
-  styles?: IStyleFunctionOrObject<IChoiceGroupOptionStyleProps, IChoiceGroupOptionStyles>;

```

Comment:
is this a breaking change?

Reply:
No, it's moving from IChoiceGroupOptionsProps to IChoiceGroupOption, and the former implements the latter, so the former will still contain it.

Reply:
One would think API extractor could perhaps do a better job reflecting this :P